### DISCUSSION TITLE
Living discussion: solver interface owners

### DISCUSSION THREAD

**Discussion by rileyjmurray** [cvxpy/cvxpy#2203] [General]
New releases of numerical solvers sometimes have regressions or API changes that break our tests. It's desirable for us to have point people ("interface owners") who can look into any such issues promptly.

We also sometimes get contributions from people out of the blue that concern existing solver interfaces. In these cases, it's desirable for the PR to be reviewed by someone who knows the solver quite well. 

We'll use this discussion to track who we should ping when one of those situations arises. To start, he's a list of solvers we support. I've tagged a couple of GitHub accounts of folks who I know are involved with the interface in question.

* [CBC](https://projects.coin-or.org/Cbc)
* [CLARABEL](https://oxfordcontrol.github.io/ClarabelDocs/), @goulart-paul, @PTNobel 
* [COPT](https://github.com/COPT-Public/COPT-Release)
* [GLOP](https://developers.google.com/optimization), @mlubin 
* [GLPK](https://www.gnu.org/software/glpk/)
* [GLPK_MI](https://www.gnu.org/software/glpk/)
* [HiGHS](https://highs.dev/), @UweGensheimer
* [OSQP](https://osqp.org/), @bstellato, @goulart-paul 
* [PROXQP](https://github.com/simple-robotics/proxsuite)
* [PDLP](https://developers.google.com/optimization), @mlubin 
* [CPLEX](https://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/)
* [NAG](https://www.nag.co.uk/nag-library-python/)
* [ECOS](https://www.embotech.com/ECOS)
* [GUROBI](https://www.gurobi.com/), @simonbowly
* [MOSEK](https://www.mosek.com/), @aszekMosek 
* [CVXOPT](http://cvxopt.org/)
* [SDPA](https://sdpa-python.github.io/), @usamamuneeb 
* [SCS](http://github.com/cvxgrp/scs), @bodono 
* [SCIP](https://scip.zib.de/)
* [XPRESS](https://www.fico.com/en/products/fico-xpress-optimization)
* [SCIPY](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.linprog.html#scipy.optimize.linprog)

I'll edit this top-level post as new info becomes available. Please comment below if you want to be added to (or removed from) this list.

**Comment by goulart-paul** [comment]
You can add me to the OSQP one as well.

**Reply by rileyjmurray** [reply]
Done!

**Comment by Transurgeon** [comment]
new entry: @UweGensheimer for HiGHS

**Comment by Transurgeon** [comment]
new entry: @simonbowly for Gurobi

**Comment by Transurgeon** [comment]
new entry: @govindchari for QOCO
